---
title: "Spatial single-cell proteomics landscape decodes the tumor microenvironmental ecosystem of intrahepatic cholangiocarcinoma "
authors:
- Libing Hong
- Jie Mei
- Xuqi Sun
- Yifan Wu
- Zhen Dong
- Yuzhi Jin
- Liaoliao Gao
- Jinlin Cheng
- Weihong Tian
- Chuan Liu
- Bin Li
- Pingping Hu
- Lulu Liu
- Shan Xin
- Xiaomeng Dai
- Peng Zhao
- Rongping Guo
- Minshan Chen
- Jingping Yun
- Bo Lin
- Wei Wei
- Weijia Fang
- Xuanwen Bao
- 林博

# Author notes (optional)
# author_notes:
#   - 'Equal contribution'  
#   - 'Equal contribution'

date: "2025-07-26T00:00:00Z"
doi: ""

# Schedule page publish date (NOT publication's date).
publishDate: "2025-05-17T00:00:00Z"

# Publication type.
# Accepts a single type but formatted as a YAML list (for Hugo requirements).
# Enter a publication type from the CSL standard.
publication_types: ["article"]

# Publication name and optional abbreviated publication name.
publication: "2025 International Conference on Intelligent Computing"
publication_short: "ICIC 2025"

abstract: The prognoses and therapeutic responses of patients with intrahepatic cholangiocarcinoma (iCCA) are influenced by spatial interactions within the tumor microenvironment (TME), yet its spatial characteristics remain understudied. This study aimed to construct a comprehensive spatial atlas of iCCA using artificial intelligence-assisted spatial multiomics approaches and identify spatial features linked to prognosis and immunotherapy. By integrating data from diverse spatial multiomics technologies—including imaging mass cytometry, spatial proteomics, spatial transcriptomics, multiplex immunofluorescence, single-cell and bulk RNA/proteomics—across over 1.06 million resolved cells, we revealed that spatial topology, encompassing cellular deposition patterns, communities, and intercellular communications, strongly correlates with patient prognosis. Notably, CD163hi M2-like resident-tissue macrophages suppress antitumor immunity through direct interaction with CD8+ T cells, associating with poorer survival. We identified five spatial subtypes with distinct prognoses and proposed subtype-specific therapeutic strategies. Additionally, a spatial TME deep learning system was developed to accurately predict patient prognosis from a single 1-mm² tumor sample. This work provides foundational insights into the spatial TME ecosystem of iCCA, laying groundwork for precise patient stratification and personalized treatment development.

# # Summary. An optional shortened abstract.
# summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags:
- CCF-C
# featured: false

# links:
# - name: Custom Link
#   url: http://example.org
# url_pdf: http://arxiv.org/pdf/1512.04133v1
# url_code: 'https://github.com/HugoBlox/hugo-blox-builder'
# url_dataset: '#'
# url_poster: '#'
# url_project: ''
# url_slides: ''
# url_source: '#'
# url_video: '#'

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  # caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/s9CC2SKySJM)'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
# projects:
# - internal-project

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
# slides: example
---
<!-- 
{{% callout note %}}
Create your slides in Markdown - click the *Slides* button to check out the example.
{{% /callout %}}

Add the publication's **full text** or **supplementary notes** here. You can use rich formatting such as including [code, math, and images](https://docs.hugoblox.com/content/writing-markdown-latex/). -->
